prov/shm/src/smr_ep.c

```diff
@@ -599,7 +599,7 @@ int smr_select_proto(enum fi_hmem_iface iface, bool use_ipc, bool cma_avail,
 		return smr_src_ipc;
 
 	if (total_len > SMR_INJECT_SIZE && cma_avail)
-		return smr_src_iov;
+		return smr_src_sar;
```

Comment:
?

Reply:
oh no... I made this change to step through SAR, and must not have noticed it got added to this PR (I thought I reset before)... need to check my diffs more carefully